src/main/java/stsjorbsmod/cards/cull/deckoftrials/Blasphemer.java

```diff
@@ -0,0 +1,53 @@
+package stsjorbsmod.cards.cull.deckoftrials;
```

Comment:
Because this is a Deck of Trials card, we'll need to update `DeckOfTrials.java`'s `reset()` to add it to the deck

Reply:
I thought I did put in in the Deck of Trials reset(). Well, regardless, it should be there in the next pr